src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache

```diff
@@ -0,0 +1 @@
+["NoCollision", "CollisionWithStationaryObject", "CollisionWithAnimal", "MultiCollision", "StationaryCamera", "ManHeldCamera", "LowQualityVideo", "LowVisibility", "PoorLighting"]
```

Comment:
don't think this should be here right? maybe add .cache files to git ignore

Reply:
Nah, it should be. If everyone commits their custom tags, then we won't end up with a bunch of duplicates

Reply:
oh thats cool, I like that idea